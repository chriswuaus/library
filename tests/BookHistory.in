BOOK HIStory 69699696969
add collection test.csv 
book history 111111
add member beans
book history 111111
rent 100000 111111
book history 111111
relinquish 100000 111111
book history 111111
add member a
add member b
add member c
add member d
add book test.csv 111111
rent 100000 111111
relinquish 100000 111111
rent 100001 111111
relinquish 100001 111111
rent 100002 111111
relinquish 100002 111111
rent 100003 111111
relinquish 100003 111111
rent 100004 111111
relinquish 100004 111111
book HISTORY 111111
exit
