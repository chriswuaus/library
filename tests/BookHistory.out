user: No such book in system.

user: 17 books successfully added.

user: No rental history.

user: Success.

user: No rental history.

user: Success.

user: No rental history.

user: Success.

user: 100000

user: Success.

user: Success.

user: Success.

user: Success.

user: Book already exists in system.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: 100000
100000
100001
100002
100003
100004

user: Ending Library process.
